readyplayerme/rpm-unreal-avatar-creator
# [SDK-317] Handle the case where the session expires

## [SDK-317](https://ready-player-me.monday.com/boards/2563815861/pulses/SDK-317)

## Description

- Subscribe to the session expiration delegate
- Fire Event when the refresh token gives 401 error
- Logout the user and Open the first available step
- Disabled precise click for the buttons in scrolling lists

## Changes

#### Fixed

-   Handle the case where the session expires.



## How to Test

-   User needs to login and then in a month open the avatar creator. In this case the user should log out automatically.

## Checklist



[SDK-317]: https://ready-player-me.atlassian.net/browse/SDK-317?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
